src/test/java/com/epam/automation/CareerTest.java

```diff
@@ -1,97 +1,94 @@
 package com.epam.automation;
 
-import com.epam.automation.enums.Menu;
-import org.openqa.selenium.By;
+import com.epam.automation.enums.MainMenu;
+import org.openqa.selenium.StaleElementReferenceException;
 import org.testng.annotations.Test;
 
-public class careerTest extends inside {
+public class CareerTest extends inside {
 
     @Test
     public void navigationTest() {
-        navigation(Menu.WHAT_WE_DO);
-        navigation(Menu.ABOUT);
-        navigation(Menu.HOW_WE_DO_IT);
-        navigation(Menu.INSIGHTS);
-        navigation(Menu.OUR_WORK);
-        navigation(Menu.CAREER);
+        navigation(MainMenu.WHAT_WE_DO_MENU);
+        navigation(MainMenu.ABOUT_MENU);
+        navigation(MainMenu.HOW_WE_DO_IT_MENU);
+        navigation(MainMenu.INSIGHTS_MENU);
+        navigation(MainMenu.OUR_WORK_MENU);
+        navigation(MainMenu.CAREER_MENU);
     }
 
     @Test
-    public void searchButtonTest() {
-        navigation(Menu.CAREER);
-        searchButton();
+    public void clickOnSearchButtonTest() {
+        navigation(MainMenu.CAREER_MENU);
+        clickOnSearchButton();
         checkingResult("Currently we are looking for");
```

Comment:
It is a good practice, to have the Assertions on this level

Reply:
checkingResult is an Assertion and navigation has it too in the method, isn't it enought or should I write more?